src/index.scss

```
@@ -4,6 +4,7 @@
 @import "avatar/index";
 @import "button/index";
 @import "forms/index";
+@import "icon";
```

I believe this should be…

```suggestion
@import "icon/index";
```